<a href="https://colab.research.google.com/github/JoshInkiriwang/Data_Science_Dicoding/blob/main/Measuring_Central_Tendency.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

**Mencari Rata-Rata dari data** (Mean)

In [ ]:
import numpy as np

nilai_siswa = np.array([80, 70, 65, 85, 90, 60, 75, 70, 80, 65])
nilai_siswa.mean()

74.0

(80, 70, 65, 85, 90, 60, 75, 70, 80, 65)

**Urutkan data dari kecil ke besar**

(60, 65, 65, 70, 70, 75, 80, 80, 85, 90)

Jumlah = 60 + 65 + 65 + 70 + 70 + 75 + 80 + 80 + 85 + 90

= 60 + 130 + 140 + 75 + 160 + 175

= 190 + 215 + 335

= 405 + 335

= 740 / 10 <= Jumlah data

Rata-Rata = 74

**Mencari nilai tengah atau persentil ke-50 dari keseluruhan observasi atau data** (Median)

In [ ]:
import numpy as np

nilai_siswa = np.array([80, 70, 65, 85, 90, 60, 75, 70, 80, 65])
np.median(nilai_siswa)

72.5

(80, 70, 65, 85, 90, 60, 75, 70, 80, 65)

**Urutkan data dari kecil ke besar**

(60, 65, 65, 70, 70, 75, 80, 80, 85, 90)

Nilai Tengahnya adalah 70 dan 75
jadinya 70+75 / 2 = 72.5

**Mencari nilai yang paling sering muncul dalam suatu data** (Mode)

In [ ]:
import numpy as np
from scipy import stats

nilai_siswa = np.array([80, 70, 65, 85, 90, 60, 75, 70, 80, 65])
modus_nilai = stats.mode(nilai_siswa)[0:2]

print(modus_nilai)

(65, 2)


(80, 70, 65, 85, 90, 60, 75, 70, 80, 65)

**Urutkan data dari kecil ke besar**

(60, 65, 65, 70, 70, 75, 80, 80, 85, 90)

*   60 = 1 Buah
*   65 = 2 Buah
*   70 = 2 Buah
*   75 = 1 Buah
*   80 = 2 Buah
*   85 = 1 Buah
*   90 = 1 Buah

Modus itu mencari angka dengan frekuensi terbanyak, dimana pada list diatas ada 3 angka yang memiliki frekuensi yang sama yaitu 2 buah. Jadinya modus pada nilai siswa tersebut ialah 65, 70, 80